sunpy/visualization/tests/test_drawing.py

```diff
@@ -31,31 +50,43 @@ def heliographic_test_map():
 
 @figure_test
 def test_draw_equator_aia171(aia171_test_map):
-    fig = plt.figure()
+    fig = Figure()
     axes = fig.add_subplot(projection=aia171_test_map)
-    aia171_test_map.plot()
+    aia171_test_map.plot(axes=axes)
     drawing.equator(axes)
+    return fig
 
 
 @figure_test
 def test_draw_prime_meridian_aia171(aia171_test_map):
-    fig = plt.figure()
+    fig = Figure()
     axes = fig.add_subplot(projection=aia171_test_map)
-    aia171_test_map.plot()
+    aia171_test_map.plot(axes=axes)
     drawing.prime_meridian(axes)
+    return fig
 
 
 @figure_test
 def test_heliographic_equator_prime_meridian(heliographic_test_map):
-    fig = plt.figure()
+    fig = Figure()
     axes = fig.add_subplot(projection=heliographic_test_map)
-    heliographic_test_map.plot()
+    heliographic_test_map.plot(axes=axes)
     drawing.equator(axes, color="blue")
     drawing.prime_meridian(axes, color="red")
+    return fig
+
+
+@figure_test
+def test_draw_extent(cropped_aia193_sample_map, euvi195_sample_map):
+    fig = Figure()
+    ax = fig.add_subplot(projection=euvi195_sample_map)
+    euvi195_sample_map.plot(axes=ax)
+    drawing.extent(ax, cropped_aia193_sample_map.wcs)
```

Comment:
Actually, this unit tests shouldn't even use a Map-based WCS since that could conceivably obscure a bug